cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectS2IAssemblyOperatorTest.java

```diff
@@ -86,6 +87,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+@SuppressWarnings({"deprecation", "HttpUrlsUsage"})
```

Comment:
Intellij warns that statements like `assertThat(connectStatus.getUrl(), is("http://foo-connect-api.test.svc:8083"))` aren't secure because it's `http` not `https`. It's probably a bit pedantic, so was an easy choice to suppress.

Reply:
Is that an actual real URL? It seems like a made-up thing in some mock .Can't it be just changed to https?